code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -159,12 +159,13 @@
 
 /datum/nanite_program/regenerative_advanced
 	name = "Bio-Reconstruction"
-	desc = "The nanites manually repair and replace organic cells, acting much faster than normal regeneration. \
```

Comment:
add a nanite sensor for interal damage

Reply:
Shouldn't the medical nanite scan also scan for internal damage though?